specification/communication/data-plane/SipRouting/preview/2022-09-01-preview/communicationservicessiprouting.json

```diff
@@ -0,0 +1,306 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "SipRouting Service",
+    "description": "SipRouting Service",
+    "contact": {
+      "email": "acsdevexdisc@microsoft.com"
+    },
+    "version": "2022-09-01-preview"
+  },
+  "paths": {
+    "/sip": {
+      "get": {
+        "tags": [
+          "SipConfiguration"
+        ],
+        "summary": "Gets SIP configuration for resource.",
+        "operationId": "GetSipConfiguration",
```

Comment:
>GetSipConfiguration [](http://example.com/codeflow?start=24&length=19)

it's recommended to use {collection}_{meaningful verb}
like SipConfiguration_Get.

Reply:
Renamed it. Thx

Reply:
@bexxx seems like this renaming of operation id is causing breaking change. I would recommend putting it back to old value so we avoid introducing breaking change.  Do you agree?

Reply:
This depends. If you have an SDK published this might affect more customers, if you just have preview versions out and not SDK, it might be fine to fix it now before this service gets more mature. All that is needed for a breaking change is the sign off in the breaking change review. so this depends on your decision, both options are fine.

Reply:
Renamed. Thank you